src/components/common/layout/WrapperClaim.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React, { useMemo } from 'react'
+import { Image, View } from 'react-native'
+import { isMobile, isMobileOnly } from 'mobile-device-detect'
+import { withStyles } from '../../../lib/styles'
+import { getScreenHeight } from '../../../lib/utils/Orientation'
+import wrapperClaimBackgroundImage from '../../../assets/wrapperClaim.svg'
+import SimpleStore from '../../../lib/undux/SimpleStore'
+Image.prefetch(wrapperClaimBackgroundImage)
+
+const sizeFactor = isMobile ? 2.7 : 1.9
+const screenHeight = getScreenHeight()
+
+const backgroundImage = {
+  backgroundImage: `url(${wrapperClaimBackgroundImage})`,
+  backgroundPosition: 'center top',
+  backgroundSize: `${screenHeight - screenHeight / sizeFactor}px`,
+  backgroundRepeat: 'no-repeat',
+  marginTop: '-3px',
+}
+
+const wrapperClaim = props => {
```

Comment:
```
const wrapperClaim =({ backgroundColor, children, style, styles, ...props }) => {
   const simpleStore = SimpleStore.useStore()
   const shouldGrow = !simpleStore.get('isMobileSafariKeyboardShown')

   consr wrapperStyles = useMemo(() => {
    const growStyle = { flexGrow: shouldGrow ? 1 : 0 }
    const backgroundStyle = backgroundColor ? { backgroundColor } : backgroundImage

    return [styles.container, backgroundStyle, growStyle, style]
   }, [shouldGrow, backgroundColor, styles.container])  

  return (
     <View data-name="viewWrapper" style={wrapperStyles} {...props}>
       {children}
     </View>
   )
 }
```

Also, pls don't  disable hook lints. Read the warning message and adjust code. If you don't understand something, ask me please

Reply:
done, thanks